build.sh

```diff
@@ -93,7 +82,7 @@ then
   fi
   OLDPATH=$PATH
   PATH=/usr/$MINGWSTR/bin:$PATH
-  CFLAGS="-mtune=pentiumpro -msse2 -O2 $CFLAGS"
+  CFLAGS="-mtune=pentiumpro -msse2 -g -O2 $CFLAGS"
```

Comment:
I added that mainly for coherence, as 99% of others systems will use these flags, so why not mingw. It can be removed of course, I don't have a very strong opinion